nixos/tests/chromium.nix

```diff
@@ -47,155 +47,218 @@ mapAttrs (channel: chromiumPkg: makeTest rec {
       xdoScript = pkgs.writeText "${name}.xdo" text;
     in "${pkgs.xdotool}/bin/xdotool '${xdoScript}'";
   in ''
+    import shlex
+    from contextlib import contextmanager, _GeneratorContextManager
+
+
     # Run as user alice
-    sub ru ($) {
-      my $esc = $_[0] =~ s/'/'\\${"'"}'/gr;
-      return "su - alice -c '$esc'";
```

Comment:
can we not hardcode the `alice` user name into the test?
If we ever change it's going to be a pain, we can use 
https://github.com/NixOS/nixpkgs/blob/1013c2e3a6f4a94305047999537c45d8853572de/nixos/tests/gnome3.nix#L29 `name`.